modules/standard/Random.chpl

```diff
@@ -1183,11 +1192,7 @@ module Random {
 
         // Fisher-Yates shuffle
         for i in 0..#D.sizeAs(D.idxType) by -1 {
-          var k = randlc_bounded(D.idxType,
```

Comment:
What are these additional changes for?

Reply:
I changed it to use getNext_noLock rather than randlc_bounded because randlc_bounded has more complex calling requirements and I wasn't convinced that it was passing the right position to randlc_bounded. But there isn't a good reason for it to track the position at all in this code so this is a simplification.